schemas/com.snowplowanalytics.monitor/application_status/jsonschema/1-0-0

```diff
@@ -0,0 +1,133 @@
+            "type": ["integer", "null"]
+          },
+          "numEventsTotal":{
+            "description": "The total number of events processed in the period to which this status applies",
+            "type": ["integer", "null"]
+          },
+          "numEventsSucceeded":{
+            "description": "The number of successfully processed events in the period to which this status applies",
+            "type": ["integer", "null"]
+          },
+          "numEventsFailed":{
+            "description": "The number of events that failed to be processed in the period to which this status applies",
+            "type": ["integer", "null"]
+          }
+        },
+        "required": ["vTracker"],
+        "additionalProperties": false
+      }
+    },
+    "numNodes": {
+      "description": "The number of nodes (instances, containers, pods, etc) on which the application is running in the period to which this status applies",
+      "type": ["integer", "null"]
+    },
+    "nodeType": {
+      "description": "The type of nodes being used, e.g. EC2, Dataflow, Pod, etc",
+      "type": ["string", "null"]
+    },
+    "nodeSpecs": {
+      "description": "Detailed specifications for the nodes being used",
+      "type": ["array", "null"],
+      "items": {
+        "description": "Each item represents a single node specification",
+        "type": "object",
+        "properties": {
+          "spec": {
+            "description": "The specification, e.g. m5.large",
+            "type": ["string", "null"]
+          },
+          "number": {
+            "description": "The number of nodes of this specification",
+            "type": ["integer", "null"]
+          },
+          "cpuCount": {
+            "description": "The number of CPUs for a node of this specification",
+            "type": ["integer", "null"]
+          },
+          "memory": {
+            "description": "The available memory for a node of this specification; in MB",
+            "type": ["integer", "null"]
+          }
+        },
+        "additionalProperties": false
+      }
+    },
+    "avgCpuUsage": {
+      "description": "The average CPU utilisation in the period to which this status applies; in %",
+      "type": ["integer", "null"],
+      "minimum": 0,
+      "maximum": 100
+    },
+    "avgMemoryUsage": {
+      "description": "The average memory usage in the period to which this status applies; in MB",
+      "type": ["integer", "null"]
+    },
+    "avgNetworkIn": {
+      "description": "The average volume of inbound network traffic in the period to which this status applies; in Bytes per second",
+      "type": ["integer", "null"]
+    },
+    "avgNetworkOut": {
+      "description": "The average volume of outbound network traffic in the period to which this status applies; in Bytes per second",
+      "type": ["integer", "null"]
```

Comment:
Some other fields we considered here:

* `last_scale_up_tstamp`
* `last_scale_down_tstamp`

Reply:
Oh, it looks like I'd missed them. I'll add them to the schema.